bot/cogs/fun.py

```diff
@@ -99,11 +101,12 @@ def __init__(self, bot: Bot):
         if unknown_emojis:
             emoji_string = ", ".join(unknown_emojis)
             # Inserts an invisible character to render mass mentions ineffective.
-            emoji_string = emoji_string.replace("@here", "@\xadhere").replace("@everyone", "@\xadeveryone")
+            emoji_string = emoji_string.replace(
+                "@here", "@\xadhere").replace("@everyone", "@\xadeveryone")
```

Comment:
Why are you changing this?

Reply:
That was due to the line being over 80 characters, which goes against PEP8:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length